packages/mom5/package.py

```diff
@@ -14,20 +14,23 @@ class Mom5(MakefilePackage):
     homepage = "https://www.access-nri.org.au"
     git = "https://github.com/ACCESS-NRI/mom5.git"
 
-    maintainers = ["harshula"]
+    maintainers("harshula", "penguian")
 
+    version("access-esm1.5", branch="access-esm1.5")
```

Comment:
The first `version()` is the default, so that has to be `master`.

Reply:
Will do.

Reply:
Done.

Reply:
I tested the default with bothe orders, and `access-esm1.5` is always the default. This is possibly because of the `".5"` at the end of the version name.

Reply:
The Spack documentation describes version comparison here: 
https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison
"Infinity versions are not automatically used when determining the latest version of a package unless explicitly required by another package or user." So, for example, if `master` is not explicitly specified, it won't be automatically used.

The Spack documentation describes version selection here: 
https://spack.readthedocs.io/en/latest/packaging_guide.html#version-selection
The largest version in the sort order is used. It is possible to mark a version as `preferred=True`.